leshan-bsserver-demo/src/main/resources/webapp/tag/bootstrap-modal.tag

```diff
@@ -80,7 +111,7 @@
 
         has_error(){
             var endpoint_has_error = (endpoint.error === undefined || endpoint.error);
-            var secMode_has_error = secMode.value === "psk" && (typeof pskId.error === "undefined" || pskId.error || typeof pskVal.error === "undefined" || pskVal.error);
+            var secMode_has_error = (secMode.value === "psk" && (typeof pskId.error === "undefined" || pskId.error || typeof pskVal.error === "undefined" || pskVal.error)) || (secMode.value === "x509" && (typeof x509Cert.error === "undefined" || x509Cert.error || typeof x509PrivateKey.error === "undefined" || x509PrivateKey.error || typeof x509ServerCert.error === "undefined" || x509ServerCert.error));
```

Comment:
Should you rewrite this to make this more readable ? using more variables or if statement ? anything which could avoid to have a line of 400 characters :)